locale/lang-tools/merge.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import argparse
+import toml
+import sys
+import os
+
+## Unmerges a file
+def unmerge(path):
+    p = path.split('/')
+    print(p)
+    print(p[-2])
+    if '-' not in p[-2]:
+        return toml.load(path)
+    p[-2] = p[-2].split('-')[0]
+    new_path = '/'.join(p)
+    if not os.path.exists(new_path):
+        return toml.load(path)
+    parent = toml.load(new_path)
+    child = toml.load(path)
+    del parent['info']
+    parent.update(child)
```

Comment:
hmm I don't like the merge system too much, because each time you add a new word in the template things will get unmerged and  we will have to manually re-merge everything

Reply:
I guess I can just call merge after `add_string.py`
Keep in mind that 203/338 (~60%) of the strings are the same in pt-pr and pt-br, and many are very similar (Seed Phrase is left in english in pt-br, while translated in pt-pt)

Reply:
if we end up with merged files after adding a new words for me it's a good solution